TrafficCapture/dockerSolution/src/main/docker/migrationConsole/setupIntegTests.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
 
-# Example usage: ./setupIntegTests.sh https://github.com/opensearch-project/opensearch-migrations.git main
+# Example usage: ./setupIntegTests.sh https://github.com/opensearch-project/opensearch-migrations.git main http://opense-clust-test.elb.us-east-1.amazonaws.com:19200 http://opense-clust-test.elb.us-east-1.amazonaws.com:9200
```

Comment:
Nit: I see that `unique_id` is a parameter as well, and it's going to be part of an index name. Can you include an example of a unique id within this example usage, and include a note to be wary of the index name syntax please?

Reply:
Have updated the usage here now